inspirehep/config.py

```diff
@@ -846,7 +846,7 @@ def _(x):
 
 # Web services and APIs
 # =====================
-BEARD_API_URL = None   # e.g. "http://beard.inspirehep.net/api"
+BEARD_API_URL = "http://beard.inspirehep.net/api"
```

Comment:
This supposed to be `None` by default as far as I know. In order for developers, Travis and so on to avoid querying the beard server every time and slowing down loading records.